### PR TITLE
Use new Numbers OAS

### PR DESCRIPTION
**Pull request by marklewin**
## Description

Use new Numbers OAS. Also changed link on docs homepage to point to the new API reference.

Please test in [review app](https://nexmo-developer-pr-1616.herokuapp.com/).

### CODE REVIEW

File: app/views/static/_products.html.erb [hunk 1]
```diff
@@ -211,7 +211,7 @@
       <nav>
         <h3 class="Vlt-grey-dark">Developer API</h3>
         <a href="/api/developer/messages">Messages</a> |
-        <a href="/api/developer/numbers">Numbers</a> |
```
Comment (mheap) [inline app/views/static/_products.html.erb]: Can we add a redirect for this?

Comment (marklewin) [reply app/views/static/_products.html.erb]: Yes, done.

**Comment by mheap** [comment]
This PR looks good, but there are a couple of things to change in the OAS to ensure that we don't lose information before merging this:

### General
Keep `api_key` and `secret` in URL but move other fields e.g. `country` and `msisdn` to body

### Search Available
type: The type of number to search for. Accepted values: landline, landline-toll-free and mobile-lvn.	

### Update a number

messagesCallbackType and messagesCallbackValue are missing. These may be missing from the list endpoint response too

**Comment by marklewin** [comment]
> Keep api_key and secret in URL but move other fields e.g. country and msisdn to body

You can only include the API key and secret in the path for the `/account/numbers` endpoint. Changed it there.

**Pull request merged**